src/hammer-vlsi/technology/sky130/sky130.tech.json

```diff
@@ -16,7 +16,7 @@
       "base var": ""
     }
   ],
-  "layer map file": "$SKY130_NDA/s8/V2.0.1/VirtuosoOA/libs/technology_library/technology_library.layermap",
+  "cadence layer map file": "$SKY130_NDA/s8/V2.0.1/VirtuosoOA/libs/technology_library/technology_library.layermap",
```

Comment:
Changing this key would break existing tech plugins? Also I don't see it changed in the schema or why this change is necessary.

Reply:
This was probably residual from when I was figuring out where a klayout file should be specified. Will change it back.